CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## main / unreleased
 
 ### Grafana Mimir
-
+* [CHANGE] Ruler: set default `evaluation-delay-duration` to 1m. #4250
```

Comment:
Please don't bundle this change with this PR.

Reply:
I pushed wrong branch, opening a new pr